app/views/users/show.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>Users#show</h1>
-<p>Details of user</p>
+<%= render partial: 'shared/header' %>
+<%= render partial: 'user', locals: {user: @user} %>
+<div class="border border-dark">
+<h2 class="fs-4">Bio</h2>
+<p>
+  <%= @user.bio %>
+</p>
+</div>
+
+
+<div>
+  <% @user_posts.each do |post| %>
+    <%= render partial:'posts/post', locals:{post: post, users: @users, comments: @comments}%>
```

Comment:
- [**_OPTIONAL SUGGESTION_**] **FINALLY**, the last part of this **`render partial`** line can be refactored as follows:

```ruby
    <%= render partial:'posts/post', locals:{post: post, users: @users, comments: post.comments}%>